docs/content/topics/operators/storage.md

```diff
@@ -9,189 +9,169 @@ aliases:
+Job's container. (Note that this may be the Job's `primaryContainer` and/or one
+or more of a Job's `sidecarContainer`(s), if applicable). This filepath value
+is assigned to the `workspaceMountPath` field on each applicable Job container.
+
+In the example `brigade.js` script below, the Job named "first-job" is
+configured with the `workspaceMountPath` value set to `/share` on both its
+primary container and its sidecar container, "sidecar". The sidecar container
+is able to read (and display) the message that the primary container prints,
+via the shared workspace mount:
+
+```javascript
+const { events, Job } = require("@brigadecore/brigadier");
+
+events.on("brigade.sh/cli", "exec", async event => {
+  let job = new Job("first-job", "debian:latest", event);
+  job.primaryContainer.command = ["bash"];
+  job.primaryContainer.arguments = ["-c", "echo 'Hello!' > /share/message"];
+  job.primaryContainer.workspaceMountPath = "/share";
+
+  job.sidecarContainers = {
+    sidecar: {
+      image: "debian:latest",
+      command: ["cat"],
+      arguments: ["/share/message"],
+      workspaceMountPath: "/share"
+    }
+  };
+
+  await job.run();
+});
+
+events.process();
 ```
 
-By default, the chart installs with persistance disabled. For various methods on enabling, as well as
-configuring other aspects of the installation, see the
-[README](https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner).
+[project file]: /topics/project-developers/projects#project-definition-files
+[10-shared-workspace example project]: https://github.com/brigadecore/brigade/blob/v2/examples/10-shared-workspace/project.yaml
 
-This chart installs a `StorageClass` named `nfs`. There are two options to configure Brigade
-to use this storage class: at the server-level (for all Brigade projects) or at the project level.
-Note that project-level settings will override the server-level settings.
+## Artemis storage
 
-In either case, there are two storage class settings:
+Brigade uses [ActiveMQ Artemis] as its messaging queue component. For more
+details on its function in Brigade, see the [Design] doc.
 
-- `cacheStorageClass`: This is used for the Job cache.
-- `buildStorageClass`: This is used for the shared per-build storage.
+Messages (i.e. work to be scheduled on the substrate) should be persisted even
+when/if Artemis itself goes down or is restarted. Therefore, by default,
+persistence is enabled via the appropriate configuration on the
+[Brigade Helm Chart][Helm chart values]. The default access mode for the
+backing PersistentVolume is `ReadWriteOnce`. This mode, although configurable,
+should not need changing.
 
-To set these at the Brigade server level, set the values below accordingly in the Brigade
-chart's `values.yaml` file:
+There are, however, other persistence options that may be useful to customize,
+such as volume size and storage class type. All configuration options can be
```

Comment:
Should we emphasize that the default storage size is very small and not adequate for production deployments?

Same Q goes for mongodb.